ArduPlane/servos.cpp

```diff
@@ -552,6 +552,10 @@ void Plane::set_servos_controlled(void)
             // manual pass through of throttle while throttle is suppressed
             SRV_Channels::set_output_scaled(SRV_Channel::k_throttle, get_throttle_input(true));
         }
+#if AP_SCRIPTING_ENABLED
+    } else if (plane.nav_scripting.current_ms > 0 && nav_scripting_enabled) {
```

Comment:
this needs the timeout check, i'd suggest making nav_scripting_active() do the timeout check and ms reset itself, with exception for AUTO in NAV_SCRIPT_TIME